taichi/ir/statements.h

```diff
@@ -217,19 +217,20 @@ class OffloadedStmt : public Stmt {
 
 class LoopIndexStmt : public Stmt {
  public:
+  Stmt *loop;
   int index;
   bool is_struct_for;

```

Comment:
I think the field `is_struct_for` would be unnecessary.

Reply:
Yes, please feel free to remove that.